src/bfx_tools/pyensembl.ml

```diff
@@ -1,13 +1,22 @@
 open Biokepi_run_environment
 open Common
 
+let set_cache_dir_command ~(run_with: Machine.t) ~reference_build =
+  let open KEDSL in
+  let env_var = "PYENSEMBL_CACHE_DIR" in
+  let cache_dir = Machine.(get_pyensembl_cache_dir run_with) in
+  let cache_dir_value = 
+    match cache_dir with
+      | Some d -> d
+      | None -> ""
```

Comment:
I think it would be better to fail with an exception there.
It's like missing tools or data, if a workflow ends up actually using it, it's better to fail at "workflow generation time".